pelican/contents.py

```diff
@@ -276,12 +288,41 @@ def replacer(m):
     def get_siteurl(self):
         return self._context.get('localsiteurl', '')
 
+    def _update_includes(self, content, source_path=None):
+        """
+            Replace {include}some.file with the
+            contents of this file.
+        """
+        regex = r"""[{|]include[|}](?P<path>[\w./]+)"""
+        hrefs = re.compile(regex, re.X)
+
+        def replacer(m):
+            path = m.group('path')
+            path = self._path_replacer(path, source_path)
+            path = posixize_path(
+                    os.path.abspath(
+                        os.path.join(self.settings['PATH'], path)
+                    )
+                )
+
+            with pelican_open(path) as text:
```

Comment:
This will throw an exception if `path` doesn't exist or isn't a file. You could use the same method done in `_update_content`. Show a warning and ignore the replace.